src/query/streams/Cargo.toml

```diff
@@ -12,24 +12,17 @@ test = false
 
 [dependencies] # In alphabetical order
 # Workspace dependencies
-common-arrow = { path = "../../common/arrow" }
 common-base = { path = "../../common/base" }
 common-datablocks = { path = "../datablocks" }
 common-datavalues = { path = "../datavalues" }
 common-exception = { path = "../../common/exception" }
-common-io = { path = "../../common/io" }
 
 # Github dependencies
 
 # Crates.io dependencies
-async-stream = "0.3.3"
-async-trait = "0.1.57"
-chrono-tz = "0.6.3"
 futures = "0.3.24"
-parking_lot = "0.12.1"
 pin-project-lite = "0.2.9"
 serde_json = { version = "1.0.85", default-features = false, features = ["preserve_order"] }
```

Comment:
Please add the appropriate comments as Xuanwo says.

We lack an effective means of determining the combination of features and I will be completing work on dependencies inheritance in the near future to avoid similar errors.